### PR TITLE
chore(github): add pull request template

### PR DESCRIPTION
**Pull request by PatrickDohn**

### CODE REVIEW

File: .github/pr_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+#### Summary
+
+-
+-
+-
+
+#### Checklist
+
+- [ ] Branch should be in format `TYPE/scope/description`
```
Comment (davidholyko) [inline .github/pr_template.md:11]: this bullet is not exactly required for this repo

File: .github/pr_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] PR title is in format `TYPE(scope): description`
+- [ ] Commits are in semantic format
```
Comment (davidholyko) [inline .github/pr_template.md:13]: this bullet is not exactly required for this repo

File: .github/pr_template.md (hunk 1 above)
Comment (PatrickDohn) [reply .github/pr_template.md]: Fixed

File: .github/pr_template.md (hunk 2 above)
Comment (PatrickDohn) [reply .github/pr_template.md]: Fixed

File: .github/pr_template.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+#### Summary
+
+-
+-
+-
+
+#### Checklist
+
+- [ ] PR title is in format `TYPE(scope): description`
+- [ ] Add link to issue
```
Comment (davidholyko) [inline .github/pr_template.md:12]: reword

"Link PR to Issue (if any issues are related"

File: .github/pr_template.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# PR Template
+
+#### Summary
+
```
Comment (davidholyko) [inline .github/pr_template.md:4]: we can add some more text to describe thes blocks

_Enumerate and describe your changes_

File: .github/pr_template.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+-
+-
+-
+
+#### Checklist
```
Comment (davidholyko) [inline .github/pr_template.md:9]: we can add some more text to describe thes blocks

_Double check list and complete these actions before merge_

File: .github/pr_template.md (hunk 3 above)
Comment (PatrickDohn) [reply .github/pr_template.md]: Fixed

File: .github/pr_template.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# PR Template
+
+#### Summary
+
+-
```
Comment (davidholyko) [inline .github/pr_template.md:5]: from what ive seen, if a person  doesnt have  that many changes, the other bullets might be still blank. maybe just one bullet is  fine and the developere can add more as they go along

Comment (PatrickDohn) [reply .github/pr_template.md:5]: Agreed, all fixed

File: .github/pr_template.md (hunk 5 above)
Comment (PatrickDohn) [reply .github/pr_template.md:9]: Fixed

File: .github/pr_template.md (hunk 4 above)
Comment (PatrickDohn) [reply .github/pr_template.md:4]: Fixed

Comment (davidholyko) [reply .github/pr_template.md:4]: this should bee on thee line directly  under ssummary

File: .github/pr_template.md (hunk 5 above)
Comment (davidholyko) [reply .github/pr_template.md:9]: should b on line directly under checklist

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**